src/main/kotlin/demo.kt

```diff
@@ -34,7 +42,19 @@ fun main() {
                     ActionData(it.clientX, it.clientY)
                 }
             }
-
+            ul {
+                seq.each().map { s: String ->
```

Comment:
Can you add an example with deletion of an specific element in the list. For example when clicking it.
In my opinion there is missing an index number or it is possible to use an object reference?

Reply:
Ok, I added my own simple example, but its not working right now... :/

Reply:
I think using an index is not a good idea. just use the reference. In the future we have to use identification via id / idProvider-interface or something like this.

Reply:
Ok, I fixed the example. My mistake. Now its working perfectly :)